lib/src/platform/web/widget_state.dart

```diff
@@ -44,7 +44,6 @@ final class MapLibreMapStateWeb extends MapLibreMapState {
             center: options.initCenter?.toLngLat(),
             bearing: options.initBearing,
             pitch: options.initPitch,
-            attributionControl: options.attribution,
```

Comment:
MapLibre GL JS has attributions enabled by default. We need to set it to `false` here so that we can use the attribution widget instead.